includes/integration.php

```diff
@@ -1024,9 +1024,15 @@ public function prepare_order_params($order, $order_merge_params = array()){
                 $purchase_params[$k] = $val;
             }
         }
-
+    
         // attach coupons data
-        $coupons_applied = $order->get_used_coupons();
+        global $woocommerce;
+        if ( version_compare( $woocommerce->version, '3.7', ">" ) ) {
+            $coupons_applied = $order->get_coupon_codes();
+        } else {
+            $coupons_applied = $order->get_used_coupons();
+        }
```

Comment:
So these two functions return the same result?

Reply:
Yes